internal/access/access_key.go

```diff
@@ -17,12 +17,16 @@ func ListAccessKeys(c *gin.Context, identityID uid.ID, name string, showExpired
 		return nil, HandleAuthErr(err, "access keys", "list", roles...)
 	}
 
-	s := []data.SelectorFunc{data.ByOptionalIssuedFor(identityID), data.ByOptionalName(name)}
+	s := []data.SelectorFunc{
+		data.Preload("IssuedForIdentity"),
+		data.ByOptionalIssuedFor(identityID),
+		data.ByOptionalName(name),
+	}
 	if !showExpired {
 		s = append(s, data.ByNotExpiredOrExtended())
 	}
 
-	return data.ListAccessKeys(db.Preload("IssuedForIdentity"), p, s...)
+	return data.ListAccessKeys(db, p, s...)
```

Comment:
In a few places I had to convert `Preload` to a selector so that I could accept a `data.GormTxn` as the first argument to the query function, and only selectors would continue to use `gorm.DB`. This is one of the places, there should be a few others.